connector/saml/types.go

```diff
@@ -115,7 +116,7 @@ type conditions struct {
 	NotBefore    xmlTime `xml:"NotBefore,attr,omitempty"`
 	NotOnOrAfter xmlTime `xml:"NotOnOrAfter,attr,omitempty"`
 
-	AudienceRestriction *audienceRestriction `xml:"AudienceRestriction,omitempty"`
+	AudienceRestriction []*audienceRestriction `xml:"AudienceRestriction,omitempty"`
```

Comment:
It's an element, not an attribute. So the XML will look like

```xml
<SubjectConfirmationData InResponseTo="foo">
  <AudienceRestriction>
  </AudienceRestriction>
  <AudienceRestriction>
  </AudienceRestriction>
</SubjectConfirmationData>
```

(with a lot more namespace annotations)